subgraph/l2/schema.graphql

```diff
@@ -0,0 +1,84 @@
+type ExampleEntity @entity {
+  id: ID!
+  count: BigInt!
+  requestId: BigInt! # uint256
+  claimId: BigInt! # uint256
+}
+type HashInvalidated @entity {
+  id: ID!
+  requestHash: Bytes! # bytes32
+  fillId: Bytes! # bytes32
+  fillHash: Bytes! # bytes32
+}
+
+type LPAdded @entity {
+  id: ID!
+  lp: Bytes! # address
+}
+
+type LPRemoved @entity {
+  id: ID!
+  lp: Bytes! # address
+}
+
+type FillManagerOwnershipTransferred @entity {
```

Comment:
Same here as above, is this the correct event name or should it be `OwnershipTransferred`?